src/components/dataset/schemaCreation/DatasetSchemaCreationView.tsx

```diff
@@ -185,7 +233,25 @@ const DatasetSchemaCreationView = withStyles(styles)(({ classes }: IProps) => {
                     Submit
                   </Button>
                 )}
-              </TabPanel>
+                {_.keys(errors).length > 0 && (
```

Comment:
Sorry, I don't think this is the best place for this comment!

I noticed that if I introduce an error to the schema definition but fix it, the error message remains:
<img width="872" alt="Screen Shot 2023-01-20 at 12 21 52 PM" src="https://user-images.githubusercontent.com/79769153/213799279-24093047-7aeb-4ce8-99c7-bb0cb0140e1d.png">

I also noticed that you can still submit a dataset creation request even if your form has errors.  Is that intentional?